spec/CodeChain-Virtual-Machine.md

```diff
@@ -80,6 +80,13 @@ Leading zeros must be truncated. Note that it is allowed to decode value with le
  1. Pop two values, first one as signature, second one as public key
  1. Verify signature via full transaction message, excluding script parameter
  1. Push true on success, false otherwise.
+* CHKMULTISIG(0x81)
+ 1. Pop one value, the value is the n in the m-of-n Multisig. The maximum value of n is 5.
```

Comment:
Do we need a limit on `n`?

Reply:
Transactions containing `CHKSIG` with large `m` will be rejected by size limit on parcel.
I think we can leave flexibility here by removing this limit.

Reply:
Isn't the limit 4MB? It's big enough to make something kind of 1000 of 1000 Multisig script.
AFAIK, ECDSA verification takes hundreds of milliseconds.

Reply:
I found the limits of stack size and script size in the spec. I think I can remove this limit then.